stable/oauth2-proxy/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 name: oauth2-proxy
-version: 3.0.0
+version: 3.1.0
```

Comment:
I'm not sure if this is the correct version to bump to. Not sure if this is breaking or not.

Reply:
It's acceptable to consider this a feature. This is not breaking (it won't break any existing installation)